lint-baseline.xml

```diff
@@ -1,5 +1,1336 @@
+        id="DeprecatedCall"
+        message="org.gradle.api.artifacts.ProjectDependency.getPath is deprecated; consider using an alternative."
+        errorLine1="    project.project(path)"
+        errorLine2="                    ~~~~">
+        <location
+            file="src/main/kotlin/com/github/jengelman/gradle/plugins/shadow/internal/GradleCompat.kt"
+            line="105"
+            column="21"/>
+    </issue>
+
+    <issue
+        id="DeprecatedCall"
+        message="org.gradle.api.artifacts.ProjectDependency.getDependencyProject is deprecated; consider using an alternative."
+        errorLine1="    dependencyProject"
+        errorLine2="    ~~~~~~~~~~~~~~~~~">
+        <location
+            file="src/main/kotlin/com/github/jengelman/gradle/plugins/shadow/internal/GradleCompat.kt"
+            line="108"
+            column="5"/>
+    </issue>
+
+    <issue
+        id="DeprecatedCall"
+        message="org.gradle.api.DefaultTask.setDescription is deprecated; consider using an alternative."
+        errorLine1="      task.description = &quot;Runs this project as a JVM application using the shadow jar&quot;"
+        errorLine2="      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~">
+        <location
+            file="src/main/kotlin/com/github/jengelman/gradle/plugins/shadow/ShadowApplicationPlugin.kt"
+            line="40"
+            column="7"/>
+    </issue>
+
+    <issue
+        id="DeprecatedCall"
+        message="org.gradle.api.DefaultTask.setDescription is deprecated; consider using an alternative."
+        errorLine1="      task.description = &quot;Runs this project as a JVM application using the shadow jar&quot;"
+        errorLine2="                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~">
+        <location
+            file="src/main/kotlin/com/github/jengelman/gradle/plugins/shadow/ShadowApplicationPlugin.kt"
+            line="40"
+            column="26"/>
+    </issue>
+
+    <issue
+        id="DeprecatedCall"
+        message="org.gradle.api.DefaultTask.setGroup is deprecated; consider using an alternative."
+        errorLine1="      task.group = ApplicationPlugin.APPLICATION_GROUP"
+        errorLine2="      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~">
+        <location
+            file="src/main/kotlin/com/github/jengelman/gradle/plugins/shadow/ShadowApplicationPlugin.kt"
+            line="41"
+            column="7"/>
+    </issue>
+
+    <issue
+        id="DeprecatedCall"
+        message="org.gradle.api.DefaultTask.setGroup is deprecated; consider using an alternative."
+        errorLine1="      task.group = ApplicationPlugin.APPLICATION_GROUP"
+        errorLine2="                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~">
+        <location
+            file="src/main/kotlin/com/github/jengelman/gradle/plugins/shadow/ShadowApplicationPlugin.kt"
+            line="41"
+            column="20"/>
+    </issue>
+
+    <issue
```

Comment:
They are not deprecated!

https://github.com/gradle/gradle/blob/8627fb2a0fac21f139e94c968be99f8642279e2c/subprojects/core-api/src/main/java/org/gradle/process/JavaExecSpec.java